components/_elements/button.tsx

```diff
@@ -2,17 +2,22 @@ interface Props {
   name?: string
   className?: string
   size?: 'small' | 'big'
+  onClick?: () => void
 }
 
-export const Button: React.FC<any> = (props: Props) => {
+export const Button: React.FC<Props> = (props: Props) => {
```

Comment:
if you typed by `React.FC<Props>`, props will be typed automatically. So `(props: Props)` can be simplified to `(props)`